tests/foreman/cli/test_user.py

```diff
@@ -1150,6 +1150,8 @@ def test_positive_create_ssh_key(self):
         :expectedresults: SSH key should be added to new user
 
         :CaseImportance: Critical
+
+        :CaseAutomation: notautomated
```

Comment:
Same here. As far as I can see, all tests in `SshKeyInUserTestCase` are stubbed, so you can add it on class level.